redash/handlers/query_results.py

```diff
@@ -187,10 +187,24 @@ def get(self, query_id=None, query_result_id=None, filetype='json'):
             query_result = get_object_or_404(models.QueryResult.get_by_id_and_org, query_result_id, self.current_org)
         elif query_id is not None:
             query = get_object_or_404(models.Query.get_by_id_and_org, query_id, self.current_org)
-
             if query is not None:
                 if settings.ALLOW_PARAMETERS_IN_EMBEDS and parameter_values:
-                    query_result = run_query_sync(query.data_source, parameter_values, query.to_dict()['query'], max_age=max_age)
+                    is_valid = True
```

Comment:
It's more complicated than that since the validation of a request is a complex condition (if we use embed key, and the signature was passed, and the signature is valid).
I think it's more readable this way.